### PR TITLE
Add a new deployment example of running Presto on a laptop

### PR DESCRIPTION
**Pull request by apc999**
Add documentation for a new deployment example in addition to docker, 
allowing users to deploy Presto to connect Hive Metastore and query S3.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by apc999** [comment]
@aweisberg per our conversation on presto site repo. Finally I get time to put some content here.

File: presto-docs/src/main/sphinx/installation/deployment.rst [hunk 1]
```diff
@@ -271,6 +276,87 @@ After launching, you can find the log files in ``var/log``:
   This is the HTTP request log which contains every HTTP request
   received by the server. It is automatically rotated and compressed.
 
+An Example Deployment on Laptop
```
Comment (zhenxiao) [inline presto-docs/src/main/sphinx/installation/deployment.rst:279]: could it be: `An Example deployment to query data in S3`?

File: presto-docs/src/main/sphinx/installation/deployment.rst [hunk 2]
```diff
@@ -271,6 +276,87 @@ After launching, you can find the log files in ``var/log``:
...
+-------------------------------
+
+This section shows how to run Presto connecting to Hive MetaStore on a single laptop to query data in an S3 bucket.
+
+Configure Hive MetaStore
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Download and extract the binary tarball of Hive.
+For example, download `apache-hive-2.3.7-bin.tar.gz <https://downloads.apache.org/hive/hive-2.3.7/apache-hive-2.3.7-bin.tar.gz>`_ .
```
Comment (zhenxiao) [inline presto-docs/src/main/sphinx/installation/deployment.rst:288]: shall we omit version?

File: presto-docs/src/main/sphinx/installation/deployment.rst [hunk 3]
```diff
@@ -271,6 +276,87 @@ After launching, you can find the log files in ``var/log``:
...
+
+.. code-block:: console
```
Comment (zhenxiao) [inline presto-docs/src/main/sphinx/installation/deployment.rst:290]: do we need to document untar commands?

File: presto-docs/src/main/sphinx/installation/deployment.rst [hunk 4]
```diff
@@ -271,6 +276,87 @@ After launching, you can find the log files in ``var/log``:
...
+    # only required for the first time
+    $ bin/schematool -dbType derby -initSchema
+
+If you want to access AWS S3, append the following lines in ``conf/hive-env.sh``.
+Hive needs the corresponding jars to access files with ``s3a://`` addresses, and AWS credentials as well to access an S3 bucket (even it is public).
+If you do not have a Hadoop installation which include above jars, download them from `maven central repository <https://repo1.maven.org/>`_.
+
+.. code-block:: console
+
+    $ export HIVE_AUX_JARS_PATH=${HADOOP_HOME}/share/hadoop/tools/lib/aws-java-sdk-core-1.10.6.jar:${HADOOP_HOME}/share/hadoop/tools/lib/aws-java-sdk-s3-1.10.6.jar:${HADOOP_HOME}/share/hadoop/tools/lib/hadoop-aws-2.8.4.jar
```
Comment (zhenxiao) [inline presto-docs/src/main/sphinx/installation/deployment.rst:312]: omit jar versions?
are these paths static and reliable?

**Review by zhenxiao — COMMENTED** [review]
am not sure whether we should add details for Hive Metastore download and start. 
Good to have documentation to deploy presto query S3 data.

File: presto-docs/src/main/sphinx/installation/deployment.rst (hunk 1 above)
Comment (apc999) [reply presto-docs/src/main/sphinx/installation/deployment.rst:279]: done

File: presto-docs/src/main/sphinx/installation/deployment.rst (hunk 2 above)
Comment (apc999) [reply presto-docs/src/main/sphinx/installation/deployment.rst:288]: done

File: presto-docs/src/main/sphinx/installation/deployment.rst (hunk 3 above)
Comment (apc999) [reply presto-docs/src/main/sphinx/installation/deployment.rst:290]: removed

File: presto-docs/src/main/sphinx/installation/deployment.rst (hunk 4 above)
Comment (apc999) [reply presto-docs/src/main/sphinx/installation/deployment.rst:312]: updated

**Comment by apc999** [comment]
@zhenxiao  thanks for reviewing my pr. I addressed your comments. please take a look.

**Review by zhenxiao — APPROVED** [review]

**Comment by zhenxiao** [comment]
LGTM @aweisberg do you have any comments?

**Pull request merged**